presto-docs/src/main/sphinx/connector/elasticsearch.rst

```diff
@@ -30,7 +30,7 @@ replacing the properties as appropriate:
     elasticsearch.default-schema=default
     elasticsearch.table-description-directory=etc/elasticsearch/
     elasticsearch.scroll-size=1000
-    elasticsearch.scroll-timeout=60000
+    elasticsearch.scroll-timeout=60000ms
```

Comment:
could we set it a different value? 60000ms is 60s, which is 1 min, too long for scroll timeout
how about we set it a smaller number, e.g. 2s?

Reply:
how about we also delete the (ms) in line 49 and line 83? it is misleading
also, in 85, the default value is 1s, not 20s, could you please fix it?

Reply:
Sure, I did it. I didn't mean to discuss if values are appropriate but the notation is wrong and example is not working. Anyway, it's good to fix those too.